jvm/src/test/scala/tastyquery/PositionSuite.scala

```diff
@@ -21,7 +23,11 @@ class PositionSuite extends BaseUnpicklingSuite(withClasses = false, withStdLib
   val imports = name"imports".singleton
 
   private def getCodePath(name: String): String =
-    s"${System.getProperty(ResourceCodeProperty)}/main/scala/${name.replace(".", "/")}.scala"
+    var Array(dir, filename) = name.split('.')
+    // A temporary workaround for path
+    // TODO: remove the workaround
+    if filename == "TraitWithSelf" then filename = "ClassWithSelf"
```

Comment:
I guess we should make it so that there is only one top level class per test file